spec/tanker/identity_spec.rb

```diff
@@ -114,7 +117,7 @@
   end
 
   describe 'parse' do
-    it 'a valid permanent identity' do
+    it 'parses a valid permanent identity' do
```

Comment:
there is a "parse" in the describe above, not sure what is common practice around here ?

Reply:
Oui, c'est encore une situation ou deux reviewers ont une opinion différente!

https://github.com/TankerHQ/identity-ruby/pull/22#discussion_r656840600

Je peut les re-enlever si tu veux.

Reply:
Idk, do what a ruby-dev expect / would do !

Reply:
Explicit it fine, I'll leave it in since at least one person thought the verb was missing.